### ISSUE TITLE
MPRIS KeyError

### ISSUE THREAD

**Issue by FichteFoll** [z411/trackma#440]
This happened yesterday for the first time. No idea how to reproduce, but I wanted to mention it at least. I also don't remember anymore whether I had to restart my player (mpv via syncplay) to make it pick up the episode.

```
Tracker (MPRIS): Initializing...
[D] Tracker (MPRIS): Enabling tracker...
Tracker (MPRIS): Using MPRIS.

Ready. Type 'help' for a list of commands.
Press tab for autocompletion and up/down for command history.
Available statuses: watching, completed, rewatching, paused, dropped, plantowatch

Tracker (MPRIS): Connecting to MPRIS player: org.mpris.MediaPlayer2.mpv
[D] Tracker (MPRIS): New playback status: Paused
[D] Tracker (MPRIS): New video: E3 2019 supercut
[D] Tracker (MPRIS): Guessing filename: E3 2019 supercut
[D] Tracker (MPRIS): Show guess: E3 2019 supercut: None
Tracker (MPRIS) warning: Found player but show not in list.
[D] Tracker (MPRIS): New tracker status: 3 (3)
[D] Tracker (MPRIS): (3) Setting active player: :1.2883
[D] Tracker (MPRIS): Starting MPRIS timer.
Tracker (MPRIS): Connecting to MPRIS player: org.mpris.MediaPlayer2.mpv.instance1748
Tracker (MPRIS): Unknown player: org.mpris.MediaPlayer2.mpd
FichteFoll [anilist] (anime) watching >> ERROR:dbus.connection:Exception in handler for D-Bus signal:
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/dbus/connection.py", line 230, in maybe_handle_message
    self._handler(*args, **kwargs)
  File "/usr/lib/python3.7/site-packages/trackma/tracker/mpris.py", line 120, in _on_update
    self._handle_status(self.statuses[sender], sender)
KeyError: ':1.3011'
ERROR:dbus.connection:Exception in handler for D-Bus signal:
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/dbus/connection.py", line 230, in maybe_handle_message
    self._handler(*args, **kwargs)
  File "/usr/lib/python3.7/site-packages/trackma/tracker/mpris.py", line 120, in _on_update
    self._handle_status(self.statuses[sender], sender)
KeyError: ':1.3011'
[D] Tracker (MPRIS): New playback status: Playing
[D] Tracker (MPRIS): New video: The Rising of the Shield Hero E23 [1080p][AAC][JapDub][GerEngSub][Web-DL].mkv
[D] Tracker (MPRIS): Guessing filename: The Rising of the Shield Hero E23 [1080p][AAC][JapDub][GerEngSub][Web-DL].mkv
[D] Tracker (MPRIS): Show guess: The Rising of the Shield Hero: {'id': 99263, 'title': 'Tate no Yuusha no Nariagari', 'my_progress': 22, 'total': 25, 'type': None, 'titles': ['Tate no Yuusha no Nariagari', 'Tate no Yuusha no Nariagari', 'The Rising of the Shield Hero', '盾の勇者の成り上がり']}
Tracker (MPRIS): Will update Tate no Yuusha no Nariagari 23
[D] Tracker (MPRIS): New tracker status: 1 (1)
[D] Tracker (MPRIS): (1) Setting active player: :1.3011
```

Version is e38d92b.

**Comment by FichteFoll** [comment]
As a quick update, I still experience this problem occasionally but I don't have steps for reproduction.

**Comment by ahmubashshir** [comment]
It occurred on vlc, isn't it?

**Comment by ahmubashshir** [comment]
It happened on vlc too, vlc single instance mode only opened the bus for a few mili seconds...
I tried to fix it on #495. The exception wasn't raised after merging to my local branch...
